pkg/operator/controller/controller_dns_configmap_test.go

```diff
@@ -643,10 +643,10 @@ func TestDesiredDNSConfigmapUpstreamResolvers(t *testing.T) {
 	}
 }
 
-// loadTestCorefile looks in the default directory of ./test-corefiles for a file matching the name argument
+// loadTestCorefile looks in the default directory of ./testdata for a file matching the name argument
 // and returns the file contents as a string.
 func loadTestCorefile(t *testing.T, name string) string {
-	corefile, err := ioutil.ReadFile("./test-corefiles/" + name)
+	corefile, err := ioutil.ReadFile("./testdata/" + name)
```

Comment:
Call `t.Helper()` at the beginning of this function. If this function errors it will print the location w.r.t. the caller. For example:

```
[aim@spicy cluster-dns-operator]$ make test
GO111MODULE=on GOFLAGS=-mod=vendor go test -race ./...
?       github.com/openshift/cluster-dns-operator/cmd/dns-operator      [no test files]
ok      github.com/openshift/cluster-dns-operator/pkg/manifests (cached)
?       github.com/openshift/cluster-dns-operator/pkg/operator  [no test files]
?       github.com/openshift/cluster-dns-operator/pkg/operator/client   [no test files]
?       github.com/openshift/cluster-dns-operator/pkg/operator/config   [no test files]
--- FAIL: TestDesiredDNSConfigmap (0.00s)
    controller_dns_configmap_test.go:63: Failed to read Corefile default_corefile: open ./testdata/default_corefile+1: no such file or directory
--- FAIL: TestDesiredDNSConfigmapUpstreamResolvers (0.00s)
    controller_dns_configmap_test.go:203: Failed to read Corefile 5upstreams: open ./testdata/5upstreams+1: no such file or directory
FAIL
FAIL    github.com/openshift/cluster-dns-operator/pkg/operator/controller       0.247s
ok      github.com/openshift/cluster-dns-operator/pkg/operator/controller/status        (cached)
?       github.com/openshift/cluster-dns-operator/pkg/util/slice        [no test files]
FAIL
make: *** [Makefile:53: test] Error 1
```

Without the helper it will just print 
```
GO111MODULE=on GOFLAGS=-mod=vendor go test -race ./...
?       github.com/openshift/cluster-dns-operator/cmd/dns-operator      [no test files]
ok      github.com/openshift/cluster-dns-operator/pkg/manifests (cached)
?       github.com/openshift/cluster-dns-operator/pkg/operator  [no test files]
?       github.com/openshift/cluster-dns-operator/pkg/operator/client   [no test files]
?       github.com/openshift/cluster-dns-operator/pkg/operator/config   [no test files]
--- FAIL: TestDesiredDNSConfigmap (0.00s)
    controller_dns_configmap_test.go:652: Failed to read Corefile default_corefile: open ./testdata/default_corefile+1: no such file or directory
--- FAIL: TestDesiredDNSConfigmapUpstreamResolvers (0.00s)
    controller_dns_configmap_test.go:652: Failed to read Corefile 5upstreams: open ./testdata/5upstreams+1: no such file or directory
FAIL
FAIL    github.com/openshift/cluster-dns-operator/pkg/operator/controller       0.220s
ok      github.com/openshift/cluster-dns-operator/pkg/operator/controller/status        (cached)
?       github.com/openshift/cluster-dns-operator/pkg/util/slice        [no test files]
FAIL
make: *** [Makefile:53: test] Error 1
```